sys/fuchsia/fidl.txt

```diff
@@ -23,9 +23,8 @@ type fidl_union_member[TAG, TYPE] {
 
 type fidl_message_header[METHOD_ORDINAL] {
 	txid		const[0, int32]
-	reserved	const[0, int32]
-	flags		const[0, int32]
-# TODO: this should be int32, but the consts passed here don't fit into 32 bits.
+	flags		array[const[0, int8], 3]
```

Comment:
I like this
```
See Header Flags for a description of the current flag definitions.
```
which points to a table with 24 "Unused" rows :)
well, actually, even 27, one byte somehow has 10 bits in it :)

Reply:
Ack.